docs/reference/objects/product/extra.md

```diff
@@ -4,52 +4,106 @@ title: Extra
 parent: Product
 ---
 
-When using a `extra` object you have access to the following attributes
+When using a `extra` object you have access to the following attributes.
+
+# extra.address
+
+Returns the [address]({% link docs/reference/objects/product/address.md %}) associated with this extra.
+
+# extra.country
+
+Returns the name of the country for the product that this extra is part of.
 
 # extra.deposit_amount
 
-Returns a humanized price for the extras deposit amount, if deposit's are not set on the extra this will just return `extra.price` e.g. `$250 Per Person`
+Returns a humanized price for the extras deposit amount, if deposit's are not set on the extra this will just return `extra.price` e.g. `$250 Per Person`.
+
+> **Deprecated**
```

Comment:
[Question] if this is deprecated should we also add that on line 17 just to be really explicit?